dallinger/recruiters.py

```diff
@@ -78,6 +78,26 @@ def open_recruitment(self, n=1):
         """
         raise NotImplementedError
 
+    def check_entry_information(self, entry_information):
```

Comment:
The contract of `check_entry_information` is a little confusing, I think. It's almost more like `normalize_entry_information` since it returns a transformation, rather than a boolean.

Reply:
Perhaps that's a better name. @pmcharrison?